KArray/KArrayCompile.lean

```diff
@@ -15,36 +15,79 @@ class Reflected (a : α)
 
 instance : Reflected (λ x => Float.sqrt x) := ⟨⟩
 
--- TODO: Somehow get a proper name
-def ename (e : Expr) : String := Id.run do
+def typeTranslationList : List (String × String) := [
+  ("Float", "double")
+]
+
+def functionTranslationList : List (String × String) := [
+  ("Float.sqrt", "sqrt")
+]
```

Comment:
The main purpose of the ReflectedType and Reflected  classes is to exactly provide these translations between Lean and C. I really want to use type classes for this, as I want the compilation be easily extensible by users, i.e. just provide one class instance and your custom C type can be used. But for now this is a sensible solution to get something working!